### PR TITLE
add go.mod and go.sum files

### PR DESCRIPTION
**Pull request by eli-nomasec**
Add a go.mod and go.sum files

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	github.com/sirupsen/logrus v1.9.3 // indirect
+	github.com/spf13/cobra v1.8.0 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	golang.org/x/net v0.22.0 // indirect
+	golang.org/x/sys v0.18.0 // indirect
+	golang.org/x/text v0.14.0 // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20240123012728-ef4313101c80 // indirect
+	google.golang.org/grpc v1.62.1 // indirect
+	google.golang.org/protobuf v1.33.0 // indirect
+)
```
Comment (siggy) [inline go.mod]: can you run `go mod tidy` to make explicit which deps are direct vs. indirect?

**Review by siggy — COMMENTED** [review]
thanks for the update (this repo needs it).

can you delete the `Gopkg.*`, as the `go.*` files supersede them?

File: go.mod (hunk 1 above)
Comment (eli-nomasec) [reply go.mod]: Done !

**Review by siggy — APPROVED** [review]
thanks!

**Pull request merged**